docs/source/faq.rst

```diff
@@ -269,6 +269,14 @@ Finally, the scheduler provides additional information about checkpointing in
 this: ``add_checkpointing_to_argparse(parser)`` adds corresponding arguments to
 the parser.
 
+
+How can I retrieve the best checkpoint obtained after tuning?
+===========================================
```

Comment:
Doesn't this === need to be as long as the line above?

Reply:
Did not know, fixed it.